tools/clang/test/HLSLFileCheck/shader_targets/mesh/mesh.hlsl

```diff
@@ -2,7 +2,8 @@
 // RUN: %dxc -E main -T ms_6_5 %s | %D3DReflect %s | FileCheck -check-prefix=REFL %s
 
 // CHECK: Mesh Shader
-// CHECK: Numthreads: (32,1,1)
+// CHECK: MeshOutputTopology=triangle
+// CHECK: NumThreads=(32,1,1)
```

Comment:
As with GS, it would be useful to have a mesh shader test that verifies all of the possible topologies. It needn't be this file though. a dedicated one might make more sense.